src/one_dragon/base/controller/pc_controller_base.py

```
@@ -188,7 +192,9 @@ def close_game(self):
         :return:
         """
         try:
-            self.game_win.get_win().close()
+            import subprocess
+            win_title = self.game_win.win_title
+            subprocess.run(["taskkill", "/f", "/fi", f"WINDOWTITLE eq {win_title}"], check=True, capture_output=True)
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)